applications/blockchainia.md

```diff
@@ -0,0 +1,123 @@
+
+Matt Dennis
+Matt is an entrepreneur with over 10 years of sales and operational experience in the Computer Security and Blockchain industry. He holds a B.s. in Computer Forensics and Information Security, and as certifications in multiple levels of the web3 tech stack. 
+
+Will Chastka
+Will has a passion for gaming, ever since his first gaming console, a sega Saturn, playing virtual fighter 2 and Shinobi Legends, to Starcraft and DOTA. His passion for crypto extends decentralized finance, gaming, and many other applications. He previously held position as Community and Marketing Manager for CyrptoAquatics NFT project, and holds a Master's in Business administration from the University of Pittsburgh and a M.S. in Business Analytics from Carnegie Mellon University. 
+
+We plan to leverage all of our experience and industry connections to build a heavily engaged community to gain market share through exploiting an emerging niche in the market with improved technology, our parachain. 
+
+### Team Code Repos
+
+- https://github.com/eca20/Coursework/tree/main/PittCoin
+- https://github.com/eca20/Coursework/tree/main/CS1699-Blockchain
+- https://github.com/eca20/BlockchainiaJRPG
+- https://github.com/eca20
+- https://github.com/bluedragonblockchain
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/edanderson4hire/
+- https://www.linkedin.com/in/wchastka/
+
+
+## Development Status :open_book:
+
+- https://github.com/eca20/BlockchainiaJRPG
+- https://github.com/eca20/SubstratePlayGround
+- https://github.com/eca20/Blockchainia-Pitch
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+- **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
+- **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```

Comment:
Please replace template text with actual values.

Reply:
updated to public :-)